static/migrations/20240219175240.php

```diff
@@ -328,7 +328,7 @@ public function process_item($item) {
 
       // Update file meta data ('fileMd5' for LiteSpeed Cache)
       $key = 'fileMd5';
-      $md5_data = $meta[$key] ?? [];
+      $md5_data = $meta[$key] ?? null;
```

Comment:
@balexey88 please check my comment about that in another repository

Reply:
Yes, this is a misspell, thanks for noticing!
The correct code is:
```php
if ( !empty($md5_data) )...
```